### ISSUE TITLE
Lists - Movie Exists - Move to Folder Option

### ISSUE THREAD

**Issue by themonk30** [Radarr/Radarr#5254]
I'm wondering if it would be possible to add an option to the lists functionality.
Basically, if the movie exists in Radarr already from the list provided - then move it to the folder selected for that list. This would be a great way to be able to prevent duplication and help organise your libraries.

E.g. Christmas List - searches Radarr library, if it finds a movie already exists from the list, it moves it from the existing directory to that selected when you create the list. If it doesn't exist, then download it as normal.

**Comment by austinwbest** [comment]
So to change the root folder of existing movies? 

That would likely not happen in an automated fashion. If you have multiple lists and the movie is in multiple lists, it would never stop moving movies. Unless I am not understanding fully.

**Comment by themonk30** [comment]
True, I see your point. Yeah effectively change the root folder of the existing movie.

I see your point about if it was on two lists....
Is there any way to search for a list of movies so that you can bulk edit them then?

**Comment by themonk30** [comment]
As a thought, a way around it - what if you could add a custom tag to everything a list finds that exists in Radarr/Sonarr?
So then you could filter a search by a tag and if you wanted too, easily manually move their root folder?

Just a thought. Then it wouldn't keep moving them between root folders but would allow you to add a tag, multiple if necessary - and the movement is down to the admin.

**Comment by themonk30** [comment]
The ability to perform

List Scan
If it exists in the library - tag it with custom tag to be filtered and bulk edited later and moved to the root folder of choice
If it doesn't exist in the library - download to the root folder as specified

Would be useful for Sonarr and Radarr please?
